### PR TITLE
ADAPT-1485: Updates to the build out.

### PR DESCRIPTION
**Pull request by sherakama**
# READY FOR REVIEW

# Summary
- Misc setup edits

# Review By (Date)
- EOD Wed 27

# Criticality
- Low

# Review Tasks

## Setup tasks and/or behavior to test

1. Review code
2. Check it out locally and try to do a build (npm run build)
3. Review preview build site

# Associated Issues and/or People
- ADAPT-1485
- @katrialesser 

# Resources
- [AMP Tool](https://stanford.levelaccess.net/index.php)
- [Accessibility Manual Test Script](https://docs.google.com/document/d/1ZXJ9RIUNXsS674ow9j3qJ2g1OAkCjmqMXl0Gs8XHEPQ/edit?usp=sharing)
- [HTML Validator](https://validator.w3.org/)
- [Browserstack](https://live.browserstack.com/dashboard) and link to [Browserstack Credentials](https://asconfluence.stanford.edu/confluence/display/SWS/External+Account+Credentials)

### CODE REVIEW

File: .codeclimate.yml [hunk 1]
```diff
@@ -28,9 +28,9 @@ plugins:
     config:
       languages:
         javascript:
-          mass_threshold: 50
+          mass_threshold: 75
```
Comment (sherakama) [inline .codeclimate.yml:31]: Found this to be a little better in most projects.

File: .codeclimate.yml [hunk 2]
```diff
@@ -46,6 +46,8 @@ plugins:
     enabled: false
   csslint:
     enabled: true
+  stylelint:
+    enabled: true
```
Comment (sherakama) [inline .codeclimate.yml:50]: Not 100% sure if this is helpful or not yet but 🤞

File: .prettierignore [hunk 3]
```diff
@@ -1,4 +0,0 @@
-.cache
```
Comment (sherakama) [inline .prettierignore:1]: Removed prettier from this package to encourage us to look for its replacement

File: gatsby-config.js [hunk 4]
```diff
@@ -1,5 +1,3 @@
-const path = require('path');
```
Comment (sherakama) [inline gatsby-config.js:1]: unused var.

File: gatsby-config.js [hunk 5]
```diff
@@ -14,7 +12,7 @@ module.exports = {
     title: `Make a Gift`,
     description: `Make a Gift to Stanford`,
     author: `Stanford University Office of Development`,
-    siteUrl: `https://makeagift.stanford.edu`,
+    siteUrl: `https://makeagift-preview.stanford.edu`,
```
Comment (sherakama) [inline gatsby-config.js:15]: Set to the preview url.

File: gatsby-node.js [hunk 6]
```diff
@@ -52,6 +53,7 @@ exports.createPages = ({ graphql, actions }) => {
             path: '/' + pagePath,
             component: storyblokEntry,
             context: {
+              slug: entry.node.full_slug,
```
Comment (sherakama) [inline gatsby-node.js:56]: Saw this in the storyblok plugin docs.

File: gatsby-node.js [hunk 7]
```diff
@@ -60,3 +62,18 @@ exports.createPages = ({ graphql, actions }) => {
...
+  loaders,
+  plugins,
+  actions,
+}) => {
+  actions.setWebpackConfig({
+    node: {
+      fs: "empty"
+    }
+  })
+}
```
Comment (sherakama) [inline gatsby-node.js:79]: Turns out the `dotenv` package makes this required and not just sass

File: netlify.toml [hunk 8]
```diff
@@ -1,7 +1,8 @@
 # BUILD Settings:
 # ##############################################################################
 [build]
-    functions = "lambda"
+  publish = "public"
```
Comment (sherakama) [inline netlify.toml:4]: This is the dir that is served up on netlify and where gatsby builds to.

File: package.json [hunk 9]
```diff
@@ -24,17 +24,13 @@
     "sanitize-html": "^1.27.5",
     "storyblok-react": "^0.1.2"
   },
-  "devDependencies": {
-    "prettier": "^2.0.5"
-  },
```
Comment (sherakama) [inline package.json:29]: We should replace this in the future.

File: src/templates/storyblok-entry.js [hunk 10]
```diff
@@ -27,9 +27,9 @@ class StoryblokEntry extends React.Component {
     let content = this.state.story.content
 
     return (
-      <div>
+      <>
```
Comment (sherakama) [inline src/templates/storyblok-entry.js:30]: One less div is probably a good thing.

**Review by yvonnetangsu — APPROVED** [review]
All looks good to me! And we'll figure some of the details out together later. GTG - thanks!

**Comment by yvonnetangsu** [comment]
@sherakama I haven't pulled it down locally - will do tomorrow. The code looks good though.

**Comment by sherakama** [comment]
Cool thanks.

**Comment by sherakama** [comment]
I'd like to clone this repo to start the alumni homesite so I am going to merge this down.

**Pull request merged**